installer/conf/omi_mapping.json

```diff
@@ -219,7 +219,8 @@
             },
             {
                 "CimPropertyName": "UsedMemory",
-                "CounterName": "Used Memory"
+                "CounterName": "Used Memory",
```

Comment:
I don't understand the use of DisplayName. Why not just change the counter name?